lang-ca.js

```diff
@@ -924,9 +930,11 @@ SnapTranslator.dict.ca = {
     'add comment':
         'afegeix un comentari',
     'undrop':
-        'recupera bloc',
+        'desfés',
```

Comment:
@bromagosa the distinction between "undrop" and "undo" has been blurred in the upcoming release. Basically "undrop" is now *almost* a full "undo", with the only limitation that it only works for blocks and scripts. I've been keeping the name "undrop" only because @brollb is working on a much, much better and way more general "undo" mechanism that truly deserves this name. For translation purposes I actually don't mind going straight to "undo" and "redo".